css/css-color/lch-001.html

```diff
@@ -3,11 +3,13 @@
 <title>CSS Color 4: Specifying Lab and LCH</title>
 <link rel="author" title="Chris Lilley" href="mailto:chris@w3.org">
 <link rel="help" href="https://drafts.csswg.org/css-color-4/#specifying-lab-lch">
-<link rel="match" href="greentext-ref.html">
+<link rel="match" href="greensquare-ref.html">
 <meta name="assert" content="lch() with no alpha">
 <style>
-    .test {color: lch(46.277% 67.945 134.427)} /* green (sRGB #008000) converted to LCH */
+    .test { background-color: red; width: 12em; height: 12em; }
+    .test { background-color: lch(46.277% 67.945 134.427); } /* green (sRGB #008000) converted to LCH */
```

Comment:
lch(46.2775% 67.9892 134.3912)

the original hue angle is only correct to 1 decimal